apps/api/src/hyp3_api/util.py

```diff
@@ -9,7 +9,10 @@ class TokenDeserializeError(Exception):
 
 
 def get_granules(jobs: list[dict]) -> set[str]:
-    return {granule for job in jobs for granule in job['job_parameters'].get('granules', [])}
+    granules = set()
+    for key in ['granules', 'reference', 'secondary']:
+        granules = granules.union({granule for job in jobs for granule in job['job_parameters'].get(key, [])})
+    return granules
```

Comment:
Can we just do this?
```suggestion
    return {
        granule
        for key in ['granules', 'reference', 'secondary']
        for job in jobs
        for granule in job['job_parameters'].get(key, [])
    }
```